client/src/components/SingleVideo.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { FaThumbsUp, FaThumbsDown } from "react-icons/fa";
+import "./SingleVideo.css";
+
+const SingleVideo = ({ video, handleDeleteVideoClick }) => {
+  const { title, url, rating, id } = video;
+  const [votes, setVotes] = useState(0); // can be 0, 1 or -1 => to add to rating when thumbsUp or ThumbsDown is clicked
```

Comment:
It's really good to include a comment explaining what values are allowed/possible when ony some values are allowed - good job!

Reply:
That said, I don't see any code that _prevents_ votes from being set to 2, or -2 - do you intend for there to be code stopping that?